pkg/compose/config.go

```diff
@@ -20,6 +23,10 @@ type (
 		AppStoreFactoryFunc func(c *Config) (AppStore, error)
 		BlockSize           int64
 		DBFilePath          string
+
+		ProxyURL   *url.URL
```

Comment:
Just not critical nitpick: the proxy related fields can be grouped into a single structure so the `Config` has a single entry/field for the proxy.